include/picongpu/param/pml.param

```diff
@@ -0,0 +1,158 @@
+/* Copyright 2019 Sergei Bastrakov
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Configure the perfectly matched layer (PML).
+ *
+ * To enable PML use YeePML field solver.
+ */
+
+#pragma once
+
+
+namespace picongpu
+{
+namespace fields
+{
+namespace maxwellSolver
+{
+namespace yeePML
+{
+
+    /* The parameters in this file are only used if field solver is YeePML.
+     * The original paper on this approach is J.A. Roden, S.D. Gedney.
+     * Convolution PML (CPML): An efficient FDTD implementation of the CFS - PML
+     * for arbitrary media. Microwave and optical technology letters. 27 (5),
+     * 334-339 (2000).
+     * https://doi.org/10.1002/1098-2760(20001205)27:5%3C334::AID-MOP14%3E3.0.CO;2-A
+     * Our implementation based on a more detailed description in section 7.9 of
+     * the book A. Taflove, S.C. Hagness. Computational Electrodynamics.
+     * The Finite-Difference Time-Domain Method. Third Edition. Artech house,
+     * Boston (2005), referred to as [Taflove, Hagness].
+     */
+
+    constexpr uint32_t THICKNESS = 8;
+
+    /** Thickness of the absorbing layer, in number of cells
+     *
+     * PML is located inside the global simulation area, near the outer borders.
+     * Setting size to 0 results in disabling absorption at the corresponding
+     * boundary. Normally thickness is between 6 and 16 cells, with larger
+     * values providing less reflections.
+     * 8 cells should be good enough for most simulations. There are no
+     * requirements on thickness being a multiple of the supercell size.
+     * It is only required that PML is small enough to fit near-boundary local
```

Comment:
see comment/question before: This is a bit unclear to me.

Reply:
Not sure which comment/question do you mean?